### PR TITLE
Migrated modules to v9

### PR DESCRIPTION
**Pull request by SerpentCS**
Migrated modules medical, medical_disease,  medical_medicament, medical_prescription to v9

### CODE REVIEW

**Comment by lasley** [comment]
Hi @SerpentCS These modules have already been converted in pending PRs #69 #72 #74 #75 & #76. Please take a look at those and let me know if I missed something :smile: 

@pedrobaeza Would you mind updating the 9.0 migration ticket for the current modules with PRs? For quick reference:
- medical
- _appointment
- _pathology
- _disease
- _disease_allergy
- medicament
- _medication
- _ medication_us
- _medical_pharmacy
- medical_pharmacy_us
- _medical_prescription
- _medical_prescription_sale

**Comment by pedrobaeza** [comment]
Done

File: medical/__init__.py [hunk 1]
```diff
@@ -20,6 +20,6 @@
 #
 ##############################################################################
 
-from . import medical_constants
-from . import models
-from . import wizard
+import medical_constants
```
Comment (lmignon) [inline medical/__init__.py:23]: you must keep the relative form for import.
from . import medical_constants

File: medical/__openerp__.py [hunk 2]
```diff
@@ -1,9 +1,12 @@
 # -*- coding: utf-8 -*-
 ##############################################################################
+#
 #    OpenERP, Open Source Management Solution
 #    Copyright (C) 2004-2009 Tiny SPRL (<http://tiny.be>). All Rights Reserved
 #    Odoo Medical, HMS Opensource Solution
+#
```
Comment (lmignon) [inline medical/__openerp__.py:7]: These cosmetic changes are useless and _pollute_ the diff

**Comment by lmignon** [comment]
@SerpentCS This PR is a duplicate of pending PRs #69 #72 #74 #75 & #76. 
@pedrobaeza IMO this one can be closed as duplicate.

**Comment by pedrobaeza** [comment]
Yeah, I think so, @SerpentCS, please check other PRS and propose possible improvements over that branches.

**Pull request closed without merging**